docs/generate_api_reference.py

```diff
@@ -91,7 +91,7 @@ def write_module_doc(module_name):
 PACKAGE_DOC_TEMPLATE = """{title}
 
 .. toctree::
-    :maxdepth: 1
+    :maxdepth: 2
```

Comment:
I'm not familiar with this -- why is maxdepth now 2?

Reply:
This controls the depth of the table of contents tree. Currently, with maxdepth = 1, the API reference page displays a list of links with two items:

* resources
* utils

Increasing maxdepth to 2 also lists out the modules under those, which makes it a lot easier to quickly scan through what's available.

* resources
   * grch37
   * grch38
   * resource_utils
* utils
   * annotations
   * constants
   * generic
   ...

Reply:
Sounds reasonable to me!